### PR TITLE
DOCS-14455 Add references for numHosts targeted in shardingStatistics

### PR DESCRIPTION
**Pull request by ianf-mongodb**

### CODE REVIEW

File: source/release-notes/4.4.txt [hunk 1]
```diff
@@ -1915,6 +1915,18 @@ output:
   - :serverstatus:`shardedIndexConsistency`
   - :serverstatus:`shardingStatistics.rangeDeleterTasks`
   - :serverstatus:`shardingStatistics.unfinishedMigrationFromPreviousPrimary`
+  - :serverstatus:`shardingStatistics.numHostsTargeted`
+
+``serverStatus`` Sharding Statistics Output Change
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CRUD Operations and Aggregations
```
Comment (jeff-allen-mongo) [inline source/release-notes/4.4.txt:1923]: I'd remove this line. It's kind of trying to act as a heading but looks a little strange in plain-text. The copy reads fine and still makes sense without this line.

File: source/reference/command/serverStatus.txt [hunk 2]
```diff
@@ -3364,6 +3397,28 @@ shardingStatistics
...
+            cluster. 
+            
+            Multi-shard operations can either be 
+            :ref:`scatter-gather<sharding-query-patterns>`
+            or shard specific. Multi-shard scatter-gather operations can
+            :ref:`consume more resources<sharding-troubleshooting-scatter-gather>`,
+            by using the 
+            :serverstatus:`shardingStatistics.numHostsTargeted`
+            metrics you can tune the aggregation queries that run on a 
+            sharded cluster.
```
Comment (jeff-allen-mongo) [inline source/reference/command/serverStatus.txt:3420]: There's some weird indentation happening here. We should be doing:

```
.. note::

   Running the :dbcommand:`serverStatus` command on...
   
```

File: source/reference/command/serverStatus.txt [hunk 3]
```diff
@@ -3364,6 +3397,28 @@ shardingStatistics
...
 
+.. serverstatus:: shardingStatistics.numHostsTargeted
+
+   Indicates the number of shards targeted for ``CRUD`` operations and 
+   aggregation commands. When a ``CRUD`` operation or aggregation
+   command is run, the following metrics will be incremented.
+
+   .. include:: /includes/fact-server-status-num-hosts-targeted.rst
+
+   .. note:: Running the :dbcommand:`serverStatus` on :binary:`mongos`
```
Comment (jeff-allen-mongo) [inline source/reference/command/serverStatus.txt:3408]: Please add "command" after "serverStatus".

File: source/reference/command/serverStatus.txt [hunk 4]
```diff
@@ -3364,6 +3397,28 @@ shardingStatistics
...
+
+   Indicates the number of shards targeted for ``CRUD`` operations and 
+   aggregation commands. When a ``CRUD`` operation or aggregation
+   command is run, the following metrics will be incremented.
+
+   .. include:: /includes/fact-server-status-num-hosts-targeted.rst
+
+   .. note:: Running the :dbcommand:`serverStatus` on :binary:`mongos` 
+            will provide insight into the aggregation operations find, 
+            insert, update, delete and aggregate that run on a sharded
```
Comment (jeff-allen-mongo) [inline source/reference/command/serverStatus.txt:3410]: Can we use a similar wording as done in the release notes and say something like:

```suggestion
            will provide insight into CRUD and aggregation operations that run on
            a sharded 
```

By saying "the aggregation operations find, insert, update, delete", it sounds like those are aggregation operators, which is a little misleading.

File: source/reference/command/serverStatus.txt [hunk 5]
```diff
@@ -3364,6 +3397,28 @@ shardingStatistics
...
+   .. note:: Running the :dbcommand:`serverStatus` on :binary:`mongos` 
+            will provide insight into the aggregation operations find, 
+            insert, update, delete and aggregate that run on a sharded 
+            cluster. 
+            
+            Multi-shard operations can either be 
+            :ref:`scatter-gather<sharding-query-patterns>`
+            or shard specific. Multi-shard scatter-gather operations can
+            :ref:`consume more resources<sharding-troubleshooting-scatter-gather>`,
+            by using the
```
Comment (jeff-allen-mongo) [inline source/reference/command/serverStatus.txt:3417]: These are two separate sentences. We should write this as:

```
Multi-shard scatter-gather operations can
:ref:`consume more resources <sharding-troubleshooting-scatter-gather>`.
By using the :serverstatus:`shardingStatistics.numHostsTargeted`
metrics, you can...
```

**Review by jeff-allen-mongo — CHANGES_REQUESTED** [review]

File: source/reference/command/serverStatus.txt (hunk 5 above)
Comment (ianf-mongodb) [reply source/reference/command/serverStatus.txt]: Updated.

File: source/reference/command/serverStatus.txt (hunk 4 above)
Comment (ianf-mongodb) [reply source/reference/command/serverStatus.txt]: Updated.

File: source/reference/command/serverStatus.txt (hunk 3 above)
Comment (ianf-mongodb) [reply source/reference/command/serverStatus.txt]: Updated.

File: source/reference/command/serverStatus.txt (hunk 2 above)
Comment (ianf-mongodb) [reply source/reference/command/serverStatus.txt]: Updated.

File: source/release-notes/4.4.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/release-notes/4.4.txt]: Updated.

**Review by jeff-allen-mongo — APPROVED** [review]

**Pull request merged**